src/services/events/events-schema.ts

```diff
@@ -12,6 +12,7 @@ export const EventValidator = z.object({
     isVirtual: z.boolean(),
     imageUrl: z.string().nullable().optional(),
     isVisible: z.boolean().default(false),
+    attendanceCount: z.number().optional(),
```

Comment:
since we've added metadata to events, these shoulnd't be visible in GET /event anymore. Can you filter these out from GET /event?

How i'd do it is:

- create a zod validator for publicEventData
- create a zod validator for internalEventData that extends public event Data
- in get /event, pass it through public event data to filter out the extra fields